generators/dockertools/templates/python/Dockerfile

```diff
@@ -15,10 +15,19 @@ RUN apk --update add --virtual build-dependencies libffi-dev openssl-dev python-
 
 COPY . /app
 <% if (!enable) { %>
+<% if (language === "PYTHON") { %>
 ENV FLASK_APP=server
+<% } else { %>
+ENV DJANGO_APP=<%= name %>
```

Comment:
Is `DJANGO_APP` a thing? Flask works oddly in that it needs environment variables, but I thought Django didn't.